src/test/java/hudson/plugins/parameterizedtrigger/test/BuildTriggerConfigTest.java

```diff
@@ -121,6 +132,65 @@ public void testGetProjectListStatic() throws Exception {
 
     }
 
+    public void testGetProjectListWithWorkflow() throws Exception {
+        Project<?, ?> masterProject = createFreeStyleProject("project");
+        WorkflowJob p = (WorkflowJob) Jenkins.getInstance().createProject(WorkflowJob.class, "workflowproject");
```

Comment:
Redundant cast. Also remember that from a `HudsonTestCase` you can use simply `jenkins` rather than `Jenkins.getInstance()`.